pasqal_cloud/batch.py

```diff
@@ -59,9 +60,25 @@ class Batch:
     jobs_count: int = 0
     jobs_count_per_status: Dict[str, int] = field(default_factory=dict)
     configuration: Optional[Union[BaseConfig, dict]] = None
+    # Ticket (#622)
+    group_id: Optional[str] = None
+    project_id: Optional[str] = None
 
     def __post_init__(self) -> None:
```

Comment:
IIRC this method is intended to parse the response from the API, so here we know that the API will return the project_id (and for now the group_id as well) so I don't think the Exception is required. Maybe I'm missing something though